ideas/122-sob-metrics.md

```diff
@@ -0,0 +1,72 @@
+## Preamble
+
+    Idea: 122-sob-metrics
+    Title: Open Bounty - Platform Usage Metrics
+    Status: Draft
```

Comment:
Is this still a draft? If so we can merge it straight away. If it is supposed to be in progress, some tweaks (marked in review) are needed